com.unity.render-pipelines.high-definition/Runtime/Lighting/LightDefinition.cs

```diff
@@ -6,7 +6,7 @@ namespace UnityEngine.Rendering.HighDefinition
 
     // Caution: Order is important and is use for optimization in light loop
     [GenerateHLSL]
-    enum GPULightType
+    enum GPULightType // Rename to "LightType"
```

Comment:
IIRC we haven't done it because LightType is already use in builtin

Reply:
This is optional. :-)